release/DOCKERHUB.md

```diff
@@ -0,0 +1,78 @@
+# Quick reference
+- Maintained by: NHS Digital
+- Where to get help: https://github.com/nhsconnect/integration-adaptor-lab-results
+- Where to file issues: https://github.com/nhsconnect/integration-adaptor-lab-results/issues
+
+# What is the Lab Results Adaptor?
+* A pre-assured implementation of Pathology and Screening FHIR adapter
+
+# How to use this image
+
+To help you begin using the Lab Results Adaptor we provide shell scripts and Docker Compose configurations.
+
+## Pre-requisites
+
+To get running make sure you have an OpenTest environment setup. The project also includes mock dependencies for local
+testing and development.
+
+## Clone the repository
+
+```bash
+git clone https://github.com/nhsconnect/integration-adaptor-lab-results.git
+```
+
+## Pull the latest changes and checkout the release tag
+
+Every tagged container on Docker hub has a corresponding tag in the git repository. Checkout the tag of the release 
+you are testing to ensure compatibility with configurations and scripts.
+
+```bash
+git pull
+git checkout 0.0.1
+```
+
+## Configure the application
+
+docker-compose.yml is preconfigured with environment values to run a standalone version of the adadptor using fake mesh.
+
+If you would like to run the adaptor against any other Mesh service or using any other database or message queue broker, modify required environment variables as described in to  [README](https://github.com/nhsconnect/integration-adaptor-lab-results/blob/0.0.1/README.md)
```

Comment:
```suggestion
If you would like to run the adaptor against any other Mesh service or using any other database or message queue broker, modify required environment variables as described in the [README](https://github.com/nhsconnect/integration-adaptor-lab-results/blob/main/README.md)
```

The linked README should be to the one in the main branch, not the release branch?